funsor/ops/builtin.py

```diff
@@ -161,12 +157,12 @@ def log_abs_det_jacobian(x, y):
 
 @Op
 def log1p(x):
-    return np.log1p(x)
+    return math.log1p(x)
 
 
 @Op
 def sigmoid(x):
-    return 1 / (1 + np.exp(-x))
+    return 1 / (1 + exp(-x))
```

Comment:
Design question: should we allow `Op`s within `Op`s (which will increase overhead of multipledispatch) or should we cleanly separate the low-level op layer from higher-level functions built using ops (e.g. @fehiepsi's higher-level gaussian math built on top of ops)?

I am leaning towards the former because it is more flexible. For example we can implement `haar_transform` and `inverse_haar_transform` as ops that use lower-level ops, but then register them as inverses.

Reply:
@eb8680 do you have an opinion?

Reply:
I think it's fine to allow `Op`s within `Op`s. It's easier to write backend-agnostic code that way, and the `multipledispatch` overhead should remain low compared to the rest of Funsor since most ops only have a couple of signatures.